index.html

```diff
@@ -577,7 +577,7 @@ <h2>Proofs (aka Signatures)</h2>
     "created": "2017-06-18T21:19:10Z",
     "creator": "https://example.com/jdoe/keys/1",
     "nonce": "c0ae1c8e-c7e7-469f-b252-86e6a0e7387e",
-    "signatureValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+
+    "proofValue": "BavEll0/I1zpYw8XNi1bgVg/sCneO4Jugez8RwDg/+
```

Comment:
I would expect this to be specific to the proof type, so `signatureValue` would still be used for a signature proof type.